### PR TITLE
TeamCity log folding

### LINKED ISSUE #107: TeamCity Foldable Log integration
CLI should detect if it's running in TeamCity the same way it detects GH Actions and Travis CI and use special instructions to fold output: https://teamcity-support.jetbrains.com/hc/en-us/community/posts/206879375-How-to-structure-output-for-build-log-so-it-is-foldable-expandable

### PR DESCRIPTION
**Pull request by edigaryev**
Resolves #107.

### CODE REVIEW

File: internal/commands/logs/teamcity.go [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+	"strings"
+)
+
+func NewTeamCityLogsRenderer(renderer *renderers.SimpleRenderer) echelon.LogRendered {
+	return &FoldableLogsRenderer{
+		delegate:          renderer,
+		startFoldTemplate: "##teamcity[blockOpened name='%s']",
+		endFoldTemplate:   "##teamcity[blockClosed name='%s']",
+		escapeFunc: func(s string) string {
+			replacer := strings.NewReplacer(
```
Comment (fkorotkov) [inline internal/commands/logs/teamcity.go:15]: You can initialize the replacer before return and pass `replacer.Replace` as `escapeFunc`.

**Review by fkorotkov — COMMENTED** [review]
Left a small not pick.

BTW do you think you can add Teamcity section to `INSTALL.md`?

File: internal/commands/logs/teamcity.go (hunk 1 above)
Comment (edigaryev) [reply internal/commands/logs/teamcity.go]: Indeed, see https://github.com/cirruslabs/cirrus-cli/pull/118/commits/1e8cd06b804809575ab557123a4465c90c4fc6a5.

**Comment by edigaryev** [comment]
> BTW do you think you can add Teamcity section to `INSTALL.md`?

See https://github.com/cirruslabs/cirrus-cli/pull/118/commits/42e01784431110336774e96acea8ffda0b4f9e81.

File: INSTALL.md [hunk 2]
```diff
@@ -106,6 +106,22 @@ jobs:
...
+```
+curl -L -o cirrus https://github.com/cirruslabs/cirrus-cli/releases/latest/download/cirrus-linux-amd64
+chmod +x ./cirrus
+./cirrus run
+```
+
+The resulting configuration should look like this:
+
+![](images/teamcity-cirrus-run-build-step-ui.png)
+
```
Comment (fkorotkov) [inline INSTALL.md:124]:
```suggestion

**Note:** you can also preinstalled the CLI on the agent itself to skip downloading it each time and just execute `cirrus run` during the step.

```

**Review by fkorotkov — COMMENTED** [review]
Let's also add the TeamCity section to the main ReadMe

**Comment by edigaryev** [comment]
> Let's also add the TeamCity section to the main ReadMe

See https://github.com/cirruslabs/cirrus-cli/pull/118/commits/f2f1a32c29a81428d400b6c607039ec6535faf69.

**Review by fkorotkov — APPROVED** [review]

**Pull request merged**